run_action.py

```diff
@@ -233,33 +247,33 @@ def main():
     existing_pull_request_comments = []
     # Request a maximum of 100 pages (3000 comments)
     for page_num in range(1, 101):
-        pull_request_comments_url = "%s/repos/%s/pulls/%s/comments?page=%d" % (
+        pull_comments_url = "%s/repos/%s/pulls/%s/comments?page=%d" % (
             github_api_url,
             repo,
             args.pull_request_id,
             page_num,
         )
-        pull_request_comments_result = requests.get(
-            pull_request_comments_url,
+        pull_comments_result = requests.get(
+            pull_comments_url,
             headers={
                 "Accept": "application/vnd.github.v3+json",
                 "Authorization": "token %s" % github_token,
             },
         )
 
-        if pull_request_comments_result.status_code != requests.codes.ok:
+        if pull_comments_result.status_code != requests.codes.ok:
             print(
                 "Request to get pull request comments failed with error code: "
-                + str(pull_request_comments_result.status_code)
+                + str(pull_comments_result.status_code)
             )
             return 1
 
-        pull_request_comments = json.loads(pull_request_comments_result.text)
+        pull_comments_chunk = json.loads(pull_comments_result.text)
 
-        if len(pull_request_comments) == 0:
+        if len(pull_comments_chunk) == 0:
             break
 
-        existing_pull_request_comments += pull_request_comments
+        existing_pull_request_comments += pull_comments_chunk
```

Comment:
I guess these are just aesthetic changes (i.e shortening the variable names)?

Reply:
Yes, and make the naming style more "uniform" in both cases.